docs/examples/composer_app/composer.json

```diff
@@ -0,0 +1,6 @@
+{
+    "require": {
+        "monolog/monolog": "1.3",
+        "marcelog/pami": "dev-master"
```

Comment:
It depends on the local installation. I usually go by https://getcomposer.org/doc/04-schema.md#minimum-stability, and make sure I use stable version whenever possible.